couchdb_custom/couchdb_custom.py

```diff
@@ -38,6 +38,13 @@ def check(self, instance):
             session.auth = (user, password)
             context = SessionContext(host, port, local_port, session)
             node_hosts = _get_couch_nodes(context)
+            for node_host in node_hosts:
+                self.gauge(
+                    'couchdb.in_maintenance_mode',
```

Comment:
nit: I'd probably just make the metric `maintenance_mode` but that doesn't really matter 😂.